packages/openapi-spec-types/README.md

```diff
@@ -48,3 +50,23 @@ See [all contributors](https://github.com/strongloop/loopback-next/graphs/contri
 # License
 
 MIT
+
+# Question
+
+Do we want multiple repos for types?
+
+option 1
+- openapi-spec-types
+- swagger-spec-types
+
+option 2
+- openapi-spec-types
+  - /v2
+  - /v3
+  
+
+Benefit of single repo:
+
+* easy to extract common suger interfaces
+* less packages to maintain, clear layout to manage different versions
+* if we only support one version, export types from one package, no need to update each dependant
```

Comment:
> IMO, we should have a single repo describing only the current (latest) version of OpenAPI spec.
>
> When a new, backwards-incompatible version of OAI is released, then we will release a new semver-major version of this package.

Let me take that back. Considering the effort needed to upgrade between OAI versions, I think it's better to have a new package for each major revision of OAI spec, because it makes it easier for us to split the work into independent chunks.